src/main/java/com/arangodb/velocypack/exception/VPackException.java

```diff
@@ -24,19 +24,19 @@
  * @author Mark Vollmary
  *
  */
-public abstract class VPackException extends RuntimeException {

```

Comment:
Leave it protected please, extend it with a concrete class if you want to customize it.